### DISCUSSION TITLE
Discuss

### DISCUSSION THREAD

**Discussion by jdev082** [JaydenDev/freecat#31] [General]
The central hub

**Comment by hello-smile6** [comment]
Yay!

**Comment by imdevarsh** [comment]
🆗

**Reply by hello-smile6** [reply]
What?

**Reply by imdevarsh** [reply]
I said, :ok:

**Reply by hello-smile6** [reply]
Do you use an emoji font on your computer by default?

**Reply by imdevarsh** [reply]
It's the macOS font, by the way

**Reply by hello-smile6** [reply]
Oh

**Comment by kccuber-scratch** [comment]
Why is there a spnosor link, it's also broken lol

**Reply by hello-smile6** [reply]
That's me, trying to make every feature github has work.
